src/test/java/StepDefinition/HotelsSD.java

```diff
@@ -2,45 +2,62 @@
 
 import DriverWrapper.Web;
 import Pages.HotelsHomepage;
+import cucumber.api.java.Before;
 import cucumber.api.java.cs.A;
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
+import org.testng.annotations.Test;
 
 public class HotelsSD {
      Web web= new Web();
     HotelsHomepage hotels= new HotelsHomepage();
 
-    @Given("I launch webpage")
-    public void launchWebpage(){
-          web.openWebpage();
-     }
+/* TC-21
+
+ */
 
   @When("I click on Sign In link")
-  public void clickOnSignIn(){
-          hotels.clickSignInLink();
-  }
+  public void clickOnSignIn(){hotels.clickSignInLink();}
   @Then("I enter incorrect email address")
-  public void enterEmailAddress(){
-        hotels.enterEmail();
-  }
+  public void enterEmailAddress(){hotels.enterEmail();}
   @And("I enter incorrect password")
-     public void enterPasswd(){
-         hotels.enterPassword();
-  }
+     public void enterPasswd(){hotels.enterPassword();}
      @Then("I click on Sign In button")
-     public void clickOnSignInButton(){
-         hotels.clickSignInButton();
-     }
+     public void clickOnSignInButton(){hotels.clickSignInButton();}
      @And("I verify error message is displayed")
-     public void errorMgsIsDisplayed(){
-         hotels.errorMsgDisplayed();
-     }
-     @And("I quit website")
-     public void quitPage(){
-         web.quitWebpage();
-     }
+     public void errorMgsIsDisplayed() {hotels.errorMsgDisplayed();}
+
+   /* TC-22
+
+    */
+
+    @Then("I click on Sign Up link")
+    public void clickOnSignUpLink(){hotels.clickSignUpLink();}
+    @Then ("I enter incorrect email address with at least '@' symbol")
+    public void enterIncorrectEmail(){hotels.enterIncorrectEmailAddress();}
+    @And("I verify error is displayed- Please check your email address")
```

Comment:
For step definition we shouldn't use @And rather it should be @Then or @When as appropriate